### DISCUSSION TITLE
primInfo != nullptr error for instanceable ArnoldAlembic prim

### DISCUSSION THREAD

**Discussion by Ark-Dem** [Autodesk/maya-usd#3542] [Q&A]
(Maya 2024.2, mayaUsd 0.26.0, Win10)

When I payload an asset with ArnoldAlembic prim into an assembly stage and make it instanceable, editing anything in this assembly returns this error:

// Error: Failed verification: ' primInfo != nullptr ' -- 

Non-instanceable state or other prim types don't seem to cause that. Does it affect anything or is just an annoyance?

assembly.usda (renaming Xform2, for example, creates the error):
```
#usda 1.0

def Xform "Xform1" (
    instanceable = true
    prepend payload = @./asset.usda@</root>
)
{
}

def Xform "Xform2"
{
}
```
asset.usda:
```
#usda 1.0
(
    defaultPrim = "root"
)

def Xform "root"
{
    def ArnoldAlembic "abc"
    {
    }
}
```

**Comment by wallworm** [comment]
Thanks for sharing. We will investigate.
